backend/edna.cabal

```diff
@@ -30,11 +30,29 @@ common common-options
 library
   import: common-options
   exposed-modules:
-      Stub
+      Edna.Config
+      Edna.Web.Server
   other-modules:
       Paths_edna
+      Edna.Web.API
+      Edna.Web.Error
+      Edna.Web.Handlers
+      Edna.Web.Types
+      Edna.Util
   hs-source-dirs:
       src
+  build-depends:
+    aeson,
+    aeson-options,
```

Comment:
`aeson-options` is [deprecated](https://github.com/serokell/aeson-options), please let's not use it, let's use [aeson-casing](https://hackage.haskell.org/package/aeson-casing) instead.

Reply:
I just wanna notice how @serokellcao and @jagajaga are just having fun together in this PR :putya: